src/app/tests/suites/certification/Test_TC_RH_2_2.yaml

```diff
@@ -52,15 +52,22 @@ tests:
       attribute: "measured value"
       PICS: A_RELATIVEHUMIDITY
       response:
+          saveAs: ValueBeforeChange
           constraints:
               type: int16
               minValue: 0
               maxValue: 10000
 
-    #Disabled as UI prompts required for user action
     - label: "Operate on device to change the relative humidity significantly"
-      disabled: true
-      PICS: MANUAL_RELATIVEHUMIDITY_CHANGE
+      PICS: MANUAL_TEMPERATURE_CHANGE
```

Comment:
Again, what does this have to do with MANUAL_TEMPERATURE_CHANGE?

Reply:
i will update it